db/migrations/202406061259_delete_content.go

```diff
@@ -19,8 +20,11 @@ var _202406061259_delete_content = &gormigrate.Migration{
 			return err
 		}
 
-		if err := tx.Exec("VACUUM").Error; err != nil {
-			return err
+		// Cannot run when testing with txdb: VACUUM must be run outside of transaction.
```

Comment:
I think we probably don't need this migration any more. Could we just comment out the content?